hal/armv7m/imxrt/10xx/105x/peripherals.h

```diff
@@ -24,6 +24,27 @@
 /* Interrupts */
 #define SIZE_INTERRUPTS 167
 
+/* DEBUG - RTT PIPE */
+#ifndef RTT_ENABLED_PLO
+#define RTT_ENABLED_PLO 0
+#endif
+
+#ifndef RTT_BUFSZ_CONSOLE_TX
```

Comment:
Maybe you don't have to define these, as `PLO_ALLDEVICES` on 105x does not include the `pipe-rtt` device, but I understand that it looks more consistent this way

Reply:
You right modified, to return error on defining and always disable.